Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '~> 1.4.0'
+gem 'cocoapods', '~> 1.5.0'
```

Comment:
can we update to the latest non-beta version of cocoapods? (1.6.x i think)

Reply:
Yeah that makes sense. I'll take a look at that